brownie/network/contract.py

```diff
@@ -239,6 +243,309 @@ def _add_from_tx(self, tx: TransactionReceiptType) -> None:
                 # if the contract self-destructed during deployment
                 pass
 
+    def get_verification_info(self) -> Dict:
+        """
+        Return a dict with flattened source code for this contract
+        and further information needed for verification
+        """
+        language = self._build["language"]
+        if language == "Vyper":
+            raise TypeError(
+                "Etherscan does not support API verification of source code "
+                "for vyper contracts. You need to verify the source manually"
+            )
+        elif language == "Solidity":
+            # Scan the AST tree for needed information
+            nodes_source = [
+                {"node": solcast.from_ast(self._build["ast"]), "src": self._build["source"]}
+            ]
+            for name in self._build["dependencies"]:
+                build_json = self._project._build.get(name)
+                if "ast" in build_json:
+                    nodes_source.append(
+                        {"node": solcast.from_ast(build_json["ast"]), "src": build_json["source"]}
+                    )
+
+            pragma_statements = set()
+            global_structs = set()
+            global_enums = set()
+            import_aliases: Dict = defaultdict(list)
+            for n, src in [ns.values() for ns in nodes_source]:
+                pragmas = n.children(filters={"nodeType": "PragmaDirective"})
+                for pragma in pragmas:
+                    src_offsets = [int(x) for x in pragma.get("src").split(":")]
+                    pragma_statements.add(src[src_offsets[0] : src_offsets[0] + src_offsets[1]])
```

Comment:
```suggestion
                    pragma_statements.add(src[slice(*pragma.offset)])
```

`solcast` node objects already expose the offset formatted in the way you require.

Reply:
I think this change can be applied in a few more places below

Reply:
Good catch, thanks!